dvc/exceptions.py

```diff
@@ -178,6 +178,11 @@ def __init__(self):
         )
 
 
+class NoMetricsParsedError(DvcException):
+    def __init__(self,):
+        super().__init__("Coul not parse any metric. Use `-v` to investigate.")
```

Comment:
@jorgeorpinel could you please review the message?

Reply:
How about something like "The metrics/plots file could not be parsed." Not sure we need the "use -v" part. What other info specific to this problem can the user get that way?

Reply:
Question: is this error output per bad metrics/plots file? Or only when all of the metrics/plots files are bad?

Reply:
How about something like "The metrics/plots file could not be parsed." for now?

Not sure we need the "use -v" part. What other info specific to this problem can the user get that way?

Reply:
@jorgeorpinel this error will only be thrown on occasion when there have been metrics found, but none could be parsed, so it should be rather rare. `-v` will allow user to see `debug` messages introduced in this change.

Reply:
If we keep the message as-is, there's a typo in `Could`

Reply:
@pmrowla fixed